### ISSUE TITLE
what does this bot do?

### ISSUE THREAD

**Issue by lorvent** [Haehnchen/crypto-trading-bot#345]
Hello,
the bot seems interesting but unfortunately i really can't understand what does it do.

i have added binance keys so it started showing up my pending orders from binance and i am able to buy and sell using it.

but can i use it to automate any trades?

is there any better docs/guide, beside readme file?

thanks.

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by Wladastic** [comment]
Yes you can use it to automate your trades, but not sure if it will still work, as I see the last commit was 10 months ago
